data.php

```diff
@@ -2,6 +2,9 @@
 require_once('./suppliers.php');

```

Comment:
Can you please rename this file to be "index.php" as this is the default file where the webserver starts from?